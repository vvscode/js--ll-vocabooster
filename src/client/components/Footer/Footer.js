import React from 'react';
import { Link } from 'react-router-dom';

import './Footer.less';

class Footer extends React.Component {
  render() {
    return (
      <div className="Footer__root">
        <Link to="/">Footer</Link>
      </div>
    );
  }
}

export default Footer;
