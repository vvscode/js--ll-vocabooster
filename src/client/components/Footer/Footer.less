@import '@client/styles/variables.css';

.Footer {
  &__root {
    background: #333;
    color: #fff;
    width: 100%;
    bottom: 0;
    left: 0;
  }

  &__container {
    margin: 0 auto;
    padding: 20px 15px;
    max-width: var(--max-content-width);
    text-align: center;
  }

  &__spacer {
    color: rgba(255, 255, 255, 0.3);
  }

  &__link {
    padding: 2px 5px;
    font-size: 1em;
    font-weight: normal;
    color: rgba(255, 255, 255, 0.6);
    text-decoration: none;

    &:hover {
      color: rgba(255, 255, 255, 1);
      font-weight: normal;
    }
  }
}
