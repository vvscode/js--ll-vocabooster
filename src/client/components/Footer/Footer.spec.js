import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer', () => {
  let props;
  let footer;

  beforeEach(() => {
    props = {
      /* List all properties with 'undefined' values */
    };

    footer = () => shallow(<Footer {...props} />);
  });

  it('renders correctly', () => {
    expect(footer()).toMatchSnapshot();
  });
});
