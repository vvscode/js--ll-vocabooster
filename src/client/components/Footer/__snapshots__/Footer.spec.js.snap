// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="Footer__root"
>
  <Link
    replace={false}
    to="/"
  >
    Footer
  </Link>
</div>
`;
